### PR TITLE
Built-in Algorithm: TensorFlow Image Classification

### PR DESCRIPTION
**Pull request by khetan2**
*Issue #, if available:*

*Description of changes:* This is a demo notebook for SageMaker Built-in Algorithm: Image Classification TensorFlow. The training dataset used in the notebook is hosted on JumpStart s3 buckets.

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `black-nb -l 100 {path}/{notebook-name}.ipynb`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,836 @@
+{
```
Comment (atqy) [inline introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb:1]: <p>I would extend the link so that it says "Amazon SageMaker <a href="https://sagemaker.readthedocs.io/en/stable/algorithms/index.html" target="_blank">Built-in Algorithms</a>" instead of just "<a href="https://sagemaker.readthedocs.io/en/stable/algorithms/index.html" target="_blank">Built-in Algorithms</a>"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb:1]: <p>I would rephrase it so that the link appears in context with the sentence instead of an awkward separate phrase i.e.  For this example, that means on an input image, predicting the class label from one of the 1000 classes of the ImageNet dataset.  -&gt; "class label from one of the 1000 classes of the ImageNet dataset" is the link to ImageNetLabels</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb:1]: <p>Looks like a Jumpstart bucket and not a bucket just dedicated to built-in algorithms? I would make it more general instead (i.e. from a public S3 bucket)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb:1]: <p>Citation is a little awkward - what information are you trying to display here (i.e. license? download link?)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb:1]: <p>I would reword the link without the number (i.e. <a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3579/#3.-Run-inference-on-the-pre-trained-model" target="_blank">Run inference on the pre-trained model</a>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (khetan2) [reply introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb:1]: <p>Done.</p>

Comment (khetan2) [reply introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb:1]: <p>done.</p>

Comment (khetan2) [reply introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb:1]: <p>done.</p>

Comment (khetan2) [reply introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb:1]: <p>Removed the citation.</p>

Comment (khetan2) [reply introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb:1]: <p>done.</p>

Comment (atqy) [reply introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb:1]: <p>Great, this looks better - one small fix, if you need to display the license, I would put it in brackets instead of another sentence (i.e. (<a href="https://jumpstart-cache-prod-us-west-2.s3-us-west-2.amazonaws.com/licenses/Apache-License/LICENSE-2.0.txt" target="_blank">Apache 2.0 License</a>))</p>

Comment (atqy) [reply introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb:1]: <p>I see, if you really want the section numbering, definately putting the word "Section" helps to clarify what the 3 is referring to. In this case, I would put "Section 3 - <a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3579/#3.-Run-inference-on-the-pre-trained-model" target="_blank">Run inference on the pre-trained model</a>" as the whole link. Without the section numbering i.e. "<a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3579/#3.-Run-inference-on-the-pre-trained-model" target="_blank">Run inference on the pre-trained model</a>" is acceptable too, either way</p>

Comment (khetan2) [reply introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb:1]: <p>done. Thanks.</p>

Comment (khetan2) [reply introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb:1]: <p>done. Thanks.</p>

**Review by atqy — APPROVED** [review]
